### PR TITLE
Update ClothingSpeedModifierSystem.cs

### PR DESCRIPTION
**Pull request by Errant-4**
## About the PR
Removed/updated Obsolete code

## Why / Balance
Project 0 Warning

## Technical details
Changed `TryGetContainingContainer` to the `Entity<T>` version
Changed `AddMarkup` to `AddMarkupOrThrow`

### CODE REVIEW

File: Content.Shared/Clothing/ClothingSpeedModifierSystem.cs [hunk 1]
```diff
@@ -76,34 +78,34 @@ private void OnClothingVerbExamine(EntityUid uid, ClothingSpeedModifierComponent
 
         var msg = new FormattedMessage();
 
-        if (walkModifierPercentage == sprintModifierPercentage)
+        if (Math.Abs(walkModifierPercentage - sprintModifierPercentage) < 1f)
         {
```
Comment (metalgearsloth) [inline Content.Shared/Clothing/ClothingSpeedModifierSystem.cs:82]: Use MathHelper.CloseTo for these.

**Review by metalgearsloth — DISMISSED** [review]

File: Content.Shared/Clothing/ClothingSpeedModifierSystem.cs (hunk 1 above)
Comment (Errant-4) [reply Content.Shared/Clothing/ClothingSpeedModifierSystem.cs]: Done

File: Content.Shared/Clothing/ClothingSpeedModifierSystem.cs [hunk 2]
```diff
@@ -46,9 +48,12 @@ private void OnHandleState(EntityUid uid, ClothingSpeedModifierComponent compone
         component.WalkModifier = state.WalkModifier;
         component.SprintModifier = state.SprintModifier;
 
+        if (!_xformQuery.TryGetComponent(uid, out var xform))
```
Comment (Plykiya) [inline Content.Shared/Clothing/ClothingSpeedModifierSystem.cs:51]: you can just use Transform(uid) to get the xform and you can assume it always has it
Transform does the query itself so it just saves on code duplication

File: Content.Shared/Clothing/ClothingSpeedModifierSystem.cs [hunk 3]
```diff
@@ -112,7 +117,10 @@ private void OnToggled(Entity<ClothingSpeedModifierComponent> ent, ref ItemToggl
         // make sentient boots slow or fast too
         _movementSpeed.RefreshMovementSpeedModifiers(ent);
 
-        if (_container.TryGetContainingContainer(ent.Owner, out var container))
+        if (!_xformQuery.TryGetComponent(ent.Owner, out var xform))
```
Comment (Plykiya) [inline Content.Shared/Clothing/ClothingSpeedModifierSystem.cs:120]: Same thing here

**Review by Plykiya — CHANGES_REQUESTED** [review]

File: Content.Shared/Clothing/ClothingSpeedModifierSystem.cs (hunk 2 above)
Comment (Errant-4) [reply Content.Shared/Clothing/ClothingSpeedModifierSystem.cs]: Already fixed on master

File: Content.Shared/Clothing/ClothingSpeedModifierSystem.cs (hunk 3 above)
Comment (Errant-4) [reply Content.Shared/Clothing/ClothingSpeedModifierSystem.cs]: Already fixed on master

**Pull request merged**